src/prefect/cli/run.py

```diff
@@ -154,7 +162,7 @@ def flow(
 
     client = Client()
     flow_id = id
-    if not flow_id:
+    if not flow_id and not version_group_id:
```

Comment:
(github won't let me comment on lines too far above this change)

You'll need to update the logic on lines 135/142 to make `version_group_id` one of the 3 ways to specify what flow to run. Right now providing `version_group_id` alone will error, but should work fine.